packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -245,7 +245,8 @@ export const usePopover = (options: PopoverOptions & PopoverInternalProps) => {
       () =>
         [
           middleware.offset !== undefined && offset(middleware.offset),
-          middleware.flip && flip({ padding: 4 }),
+          // TODO: Temporary solution that sets flip padding to one higher than size padding to prefer best fit.
+          middleware.flip && flip({ padding: 5 }),
```

Comment:
This comment is not accurate or even useful.

- "best fit" is too vague; maybe we can mention the issue as well.
- I don't consider to be a "temporary solution" because that would imply we plan to come back to this (which we don't, _unless_ we find other issues).

Something like the following would be more appropriate:

```suggestion
          middleware.flip && flip({ padding: 5 }), // one higher than `size` padding to prefer best fit (see #2597)
```